### DISCUSSION TITLE
Why do some of the install commands end with "::*"

### DISCUSSION THREAD

**Discussion by StingyJack** [dotnet/templating#2973] [General]
on [this](https://github.com/dotnet/templating/wiki/Available-templates-for-dotnet-new) page?

I get the wildcarding but not the `::` on some of them. I cant search for it here because GH strips out special characters.

**Comment by vlada-shubina** [comment]
hi @StingyJack 

`dotnet new` uses the following syntax with `-i` `--install` option: `<NuGet package ID>::<version>`
For example: `dotnet new -i Microsoft.DotNet.Common.ProjectTemplates.5.0::5.0.0` 

To install the latest version you can use either just package ID or add `::*`. Both works the same
-  `dotnet new -i Microsoft.DotNet.Common.ProjectTemplates.5.0::*` 
-  `dotnet new -i Microsoft.DotNet.Common.ProjectTemplates.5.0`
